authentication/auth.js

```diff
@@ -4,52 +4,84 @@ const JsonWebToken = require('jsonwebtoken');
 const Bcrypt = require('bcrypt');
 class Auth {
 
+    /**
+     * Create Account
+     * add new account in data base
+     * @param {*} req request data 
+     * @param {*} res responce result 
+     */
     register(req, res) {
-        // console.log(couchbaseCollection, 'dsfdsfdss');
-        if (!req.body.email) {
-            return res.status(401).send({ "message": "An `email` is required" })
-        } else if (!req.body.password) {
-            return res.status(401).send({ "message": "A `password` is required" })
-        }
+        
         let id = UUID.v4();
         const account = {
             "User_Id": id,
             "email": req.body.email,
-            "password": Bcrypt.hashSync(req.body.password, 10)
+            "password": Bcrypt.hashSync(req.body.password, 10),
+            "wallet" : 200000,
+            "betAmount" : 100 ,
+            "freeSpin"  : 0,
+            "WinFreeSpinAmount" : 0
         }
-        couchbaseCollection.insert(req.body.email, account, (error, result) => {
-            if (error) {
-                return res.status(500).send({});
+        couchbaseCollection.get(req.body.email,account,(err,reslt)=>{
+            if(reslt){
+                return res.status(401).send({ "success": false, "message": "This `Email Id` exists" });
+            }else{
+                couchbaseCollection.insert(req.body.email, account, (error, result) => {
+                    if (error) {
+                        return res.status(500).send({});
+                    }
+                    res.send({"message" : "You have been registered successfully"});
+                });
             }
-            res.send(result);
-        });
+        })
+        
     }
 
-    login(req,res) {
-        if (!req.body.email) {
-            return res.status(401).send({ "message": "An `email` is required" })
-        } else if (!req.body.password) {
-            return res.status(401).send({ "message": "A `password` is required" })
-        }
+    /**
+     * login
+     * @param {request} req request data 
+     * @param {Response} res response Token
+     */
+    login(req, res) {
 
         couchbaseCollection.get(req.body.email,(error,account) =>{
-            console.log(account.content.User_Id);
-            if(error) {
-                return response.status(500).send({ code: error.code, message: error.message });
+            
+            if(!account) {
+                return res.status(500).send({"message": "User not foundPlease register first You are not registered"});
```

Comment:
"User not found" OR "Please register first" OR "You are not registered"
Use only one message from above